src/states/florida/main.py

```diff
@@ -1,31 +1,33 @@
 from bs4 import BeautifulSoup
 from tqdm import tqdm
-
-from common import cache_request, decode_email
+from common import init_selenium_driver, cache_selenium
 
 BASE_URL = 'https://dos.elections.myflorida.com/supervisors/'
 
 
-def parse_county(soup):
+def fetch_and_parse_county(county_url, driver):
+  html = cache_selenium(county_url, wait=1, driver=driver)
+  soup = BeautifulSoup(html, 'html.parser')
   county = soup.find('p', class_='title').text.split('Supervisor')[0].strip()
   links = soup.find(id='rightContent')('a')
   return {
     'locale': county,
     'official': soup.find('span', class_='bigRed').text.replace(u'\xa0', ' ').split(',')[0].strip(),
-    'emails': [decode_email(links[0].find('span', class_='__cf_email__').get('data-cfemail')).strip()],
+    'emails': [links[0]['href'].replace('mailto:', '').strip()],
     'url': links[1]['href'].strip(),
     'county': county,
   }
 
 
-def fetch_data():
-  data = []
-  text = cache_request(BASE_URL)
-  soup = BeautifulSoup(text, 'html.parser')
-  for county_link in tqdm(soup.select('a[href^=countyInfo]')):
-    text = cache_request(BASE_URL + county_link['href'])
-    data.append(parse_county(BeautifulSoup(text, 'html.parser')))
-
+def fetch_data(verbose=True):
+  driver = init_selenium_driver()  # will be using repeatedly
```

Comment:
This is fine -- in the future, you may want to consider using a [context manager](https://book.pythontips.com/en/latest/context_managers.html) for cases where you want to always close.  [Context Lib](https://docs.python.org/3/library/contextlib.html) is the preferred way to write one of these.